pkg/cast/cast.go

```diff
@@ -905,14 +913,22 @@ func ToUint64Slice(input interface{}, sn Strictness) ([]uint64, error) {
 	return result, nil
 }
 
-func ToFloat64Slice(input interface{}, sn Strictness) ([]float64, error) {
+func ToFloat64Slice(input interface{}, sn Strictness, anc ARRAY_NIL_CONVERT) ([]float64, error) {
```

Comment:
Shall we make similar changes to other ToXXXSlice functions?

Reply:
Those methods don't seem to affect the aggregation function. Should I modified them ? maybe in a new pull request .